kiwi/utils/command_capabilities.py

```diff
@@ -103,10 +103,11 @@ def check_version(
         try:
             command = Command.run(arguments)
             for line in command.output.splitlines():
-                match = re.search('[0-9]+(\.[0-9]+)*', line)
-                if match:
+                matches = re.findall(r'([0-9]+(\.[0-9]+)*)', line)
+                if matches:
+                    match = max(matches)[0]
```

Comment:
I'm not too fluent in python - does this do a length comparison and not a lexical sort?

Reply:
Good point over tuples of strings this is no clear which criteria is being used. I am fixing it.

Reply:
Just checked, it's a lexical sort:

```
max(["z", "aa"])
'z'
```

So probably not working as desired?

Reply:
Finds the maximum, given that the re.findall() returns as list it will pick the item in the list that has the "highest" value. The way the regular expression is written that means max() will find first the tuple in the list that has the most populated entries and if there are multiple tuples with the same number of entries it will pick the one that has the highest numerical value in the first entry in the tuple, then compare the second entry and so on. It's a length first approach, yes. The way the regex is written it should be that way.

Reply:
@rjschwei The issue is that `"aaaaa" < "z"`, so the length didn't actually have an influence at all.

That's fixed now though.